core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-editor-modal.controller.spec.ts

```diff
@@ -255,7 +256,23 @@ describe('Question Suggestion Editor Modal Controller', function() {
         .toHaveBeenCalled();
     });
 
+    it('should fail to update the question' +
```

Comment:
Nit: Can we add word to this line until the line limit?

Reply:
done